Update punteggiatura-e-grammatica.rst (#31)
italia/writing-toolkit: suggerimenti-di-scrittura/punteggiatura-e-grammatica.rst

Punteggiatura e grammatica
==========================

Apostrofo
-------------

> Lâ€™apostrofo indica unâ€™elisione e non va mai separato dalle parole che lo precedono e lo seguono. Lâ€™apostrofo non deve essere usato al posto

Reviewer: ```suggestion
L’apostrofo indica un’elisione e non va mai separato dalle parole che lo precedono e lo seguono. L’apostrofo non deve essere usato al posto
```

> dell’accento nelle lettere accentate (es. è, È ecc.) perché possono produrre un’errata lettura da parte dei lettori di schermo (_screen reader_).

Reviewer: ```suggestion
dell’accento nelle lettere accentate (es. è, È ecc.) perché possono produrre un’errata lettura da parte dei lettori di schermo (_screen reader_).

```

> dellâ€™accento nelle lettere accentate (es. Ã¨, Ãˆ ecc.) perché possono produrre unâ€™errata lettura da parte dei lettori di schermo (_screen reader_).

Reviewer: ```suggestion
dell’accento nelle lettere accentate (es. è, È ecc.) perché possono produrre un’errata lettura da parte dei lettori di schermo (_screen reader_).
```

.. admonition:: use

>          puÃ²

Reviewer: ```suggestion
         può
```

         
.. admonition:: use-not

         puo' 

|

Esempio o es.
-------------

> Nel testo Ã¨ meglio scriverlo per esteso: â€œper esempioâ€. Se lo usi tra parentesi, puoi scegliere la forma abbreviata (es. come in questo caso). Preferire lâ€™uso di â€œper esempioâ€allâ€™uso di: â€œad esempioâ€.

Reviewer: ```suggestion
Nel testo è meglio scriverlo per esteso: “per esempio”. Se lo usi tra parentesi, puoi scegliere la forma abbreviata (es. come in questo caso). Preferire l’uso di “per esempio” all’uso di: “ad esempio”.
```

|

Incisi (parentesi tonde o trattino lungo)
---------------------

> Per staccare dal resto del testo una frase  Ã¨ preferibile usare le parentesi tonde che delimitano con maggiore forza lâ€™inciso. Allâ€™esterno devono sempre essere precedute e seguite da uno spazio per isolarne il contenuto, eccetto quando la parentesi chiusa Ã¨ seguita da una virgola o da un punto. Dopo che la parentesi è stata aperta e prima che sia chiusa, invece, non va inserito lo spazio. Si utilizzano principalmente per inserire chiarimenti che spezzano la linearità del discorso. 

Reviewer: ```suggestion
Per staccare dal resto del testo una frase è preferibile usare le parentesi tonde che delimitano con maggiore forza l”inciso. All”esterno devono sempre essere precedute e seguite da uno spazio per isolarne il contenuto, eccetto quando la parentesi chiusa è seguita da una virgola o da un punto. Dopo che la parentesi è stata aperta e prima che sia chiusa, invece, non va inserito lo spazio. Si utilizzano principalmente per inserire chiarimenti che spezzano la linearità del discorso. 
```

> A volte viene utilizzata anche la trattino lungo o disgiuntivo (—) che  Ã¨ un segno della punteggiatura diverso dal trattino corto o di unione (–) e, dal punto di vista tipografico,  Ã¨ lunga circa il doppio rispetto al trattino. Il trattino lungo viene usato, per esempio, come indicatore grafico per marcare lâ€™apertura del discorso diretto. Deve sempre essere preceduto e seguito da uno spazio e, nel caso in cui lâ€™inciso termini a fine periodo, va omesso il trattino di chiusura. Si noti anche che il tratto corto (–) viene di solito usato nei nomi composti (ad esempio e-commerce) e, di norma, non  Ã¨ preceduto o seguito da spazi.

Reviewer: ```suggestion
A volte viene utilizzata anche la trattino lungo o disgiuntivo (—) che è un segno della punteggiatura diverso dal trattino corto o di unione (–) e, dal punto di vista tipografico, è lunga circa il doppio rispetto al trattino. Il trattino lungo viene usato, per esempio, come indicatore grafico per marcare lapertura del discorso diretto. Deve sempre essere preceduto e seguito da uno spazio e, nel caso in cui l’inciso termini a fine periodo, va omesso il trattino di chiusura. Si noti anche che il tratto corto (–) viene di solito usato nei nomi composti (ad esempio e-commerce) e, di norma, non è preceduto o seguito da spazi.
```

In generale usa iniziali minuscole, tranne quando si tratta di nomi propri o di eccezioni come:

> -  titoli istituzionali completi, seguiti dal nome proprio (il Ministro per le Politiche agricole Nome Cognome);

Reviewer: Questo credo sia da lasciare in minuscolo:

* https://it.wikipedia.org/wiki/Ministero_delle_politiche_agricole_alimentari_e_forestali
* https://docs.italia.it/italia/designers-italia/writing-toolkit/it/bozza/le-parole-della-pubblica-amministrazione/m.html?highlight=ministero

Author: Si può trovare molta disomogeneità sul web (e non solo): è diffuso sia l'uso di tutte le prime lettere in maiuscolo (https://www.mef.gov.it/index.html, https://www.mite.gov.it/, https://www.esteri.it/it/), che quello di tutte le lettere in minuscolo (https://www.mise.gov.it/index.php/it/). 
L'iniziale degli organi (es. Ministero, Consiglio ecc.) è di norma maiuscola, minuscola se usata in modo generico (Manuale di stile). Anche l'iniziale della prima parola che individua l'ambito di competenza (Ministero della Difesa, Ministero della Salute ecc.) è maiuscola, questo consente (senza abusare delle maiuscole come richiesto già dalla Direttiva Frattini) di articolare un elenco formato da più ministeri, distinguendo così dove finisce una e comincia l'altra e di agevolare la lettura dei testi (guidando l'occhio) soprattutto quando i nomi dei ministeri sono molto articolati. Ad es. I Ministeri dell'Ambiente e della tutela del territorio e del mare, del Lavoro e delle politiche sociali e delle Infrastrutture e della mobilità... In termini di accessibilità, inoltre, questa scelta si presenta come un valore aggiunto, basta pensare alle persone con difficoltà di apprendimento e disabilità cognitive.  Al riguardo avrei preparato anche le modifiche alla lettera M per ministro e ministero (sarebbe in una delle prossime pull). Che ne pensi?

   
   3. e per il terzo: se è l’ultimo punto si concluderà con il punto fermo.
   
   |
   
Puntini sospensivi
------------------

Il numero di puntini da utilizzare è sempre e solo tre (…). Si usano per indicare che un elenco può continuare o per lasciare un discorso in sospeso. Quando si vuole indicare l’omissione di una parte di un testo citato, vanno usati tra parentesi. Quando i punti chiudono il discorso non bisogna aggiungere il punto fermo e la frase successiva inizia con la lettera maiuscola; in un elenco che termina con i puntini di sospensione non si fanno precedere dalla congiunzione “e”; in un elenco che termina con i puntini di sospensione non si deve aggiungere la parola “eccetera” in quanto avendo il medesimo significato e sarebbe una ripetizione. 

> 
> 

Reviewer: ```suggestion
```

> Il numero di puntini da utilizzare Ã¨ sempre e solo tre (…). Si usano per indicare che un elenco può continuare o per lasciare un discorso in sospeso. Quando si vuole indicare lâ€™omissione di una parte di un testo citato, vanno usati tra parentesi. Quando i punti chiudono il discorso non bisogna aggiungere il punto fermo e la frase successiva inizia con la lettera maiuscola; in un elenco che termina con i puntini di sospensione non si fanno precedere dalla congiunzione â€œeâ€; in un elenco che termina con i puntini di sospensione non si deve aggiungere la parola â€œecceteraâ€ in quanto avendo il medesimo significato e sarebbe una ripetizione. 

Reviewer: ```suggestion
Il numero di puntini da utilizzare è sempre e solo tre (…). Si usano per indicare che un elenco può continuare o per lasciare un discorso in sospeso. Quando si vuole indicare l’omissione di una parte di un testo citato, vanno usati tra parentesi. Quando i punti chiudono il discorso non bisogna aggiungere il punto fermo e la frase successiva inizia con la lettera maiuscola; in un elenco che termina con i puntini di sospensione non si fanno precedere dalla congiunzione “e”; in un elenco che termina con i puntini di sospensione non si deve aggiungere la parola “eccetera” in quanto avendo il medesimo significato e sarebbe una ripetizione. 
```
